src/main/java/com/marginallyclever/makelangelo/Translator.java

```diff
@@ -92,18 +92,15 @@ static private boolean doesLanguagePreferenceExist() throws BackingStoreExceptio
 	 * save the user's current language choice
 	 */
 	public static void saveConfig() {
-		logger.debug("Translator::saveConfig()");
+		logger.debug("saveConfig()");
```

Comment:
we have many saveConfig().  the new message is more ambiguous.

Reply:
When a log is printed, the package+class in printed like this: 
```
2021-12-30 22:06:51,618 DEBUG com.marginallyclever.makelangelo.Translator - saveConfig
```
or in dev:
```
2021-12-30 22:06:51,618 DEBUG c.m.makelangelo.Translator - saveConfig
```

so you know where it came from.